[2023-10-17T11:19:12.539+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.549+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-17T11:19:12.549+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-17T11:19:12.562+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-17 00:00:00+00:00
[2023-10-17T11:19:12.570+0000] {standard_task_runner.py:57} INFO - Started process 577 to run task
[2023-10-17T11:19:12.573+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-17T00:00:00+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpqwce8o2d']
[2023-10-17T11:19:12.574+0000] {standard_task_runner.py:85} INFO - Job 80: Subtask waiting_for_data
[2023-10-17T11:19:12.622+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [running]> on host 238c1d4ff767
[2023-10-17T11:19:12.706+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-17T00:00:00+00:00'
[2023-10-17T11:19:12.714+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-17T11:19:12.715+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-17T11:19:12.715+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-17T11:19:12.715+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-17T11:19:12.722+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210317T000000, start_date=20231017T111912, end_date=20231017T111912
[2023-10-17T11:19:12.745+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-17T11:19:12.759+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T10:42:41.548+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-18T10:42:41.559+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-18T10:42:41.560+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T10:42:41.598+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-17 00:00:00+00:00
[2023-10-18T10:42:41.606+0000] {standard_task_runner.py:57} INFO - Started process 1540 to run task
[2023-10-18T10:42:41.609+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-17T00:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpekmdge7n']
[2023-10-18T10:42:41.610+0000] {standard_task_runner.py:85} INFO - Job 79: Subtask waiting_for_data
[2023-10-18T10:42:41.670+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [running]> on host 2473b23ba069
[2023-10-18T10:42:41.741+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-17T00:00:00+00:00'
[2023-10-18T10:42:41.756+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 257, in execute
    raise e
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/base.py", line 239, in execute
    poke_return = self.poke(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/sensors/filesystem.py", line 61, in poke
    hook = FSHook(self.fs_conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/filesystem.py", line 38, in __init__
    conn = self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 72, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 477, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `fs_default` isn't defined
[2023-10-18T10:42:41.760+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210317T000000, start_date=20231018T104241, end_date=20231018T104241
[2023-10-18T10:42:41.774+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 79 for task waiting_for_data (The conn_id `fs_default` isn't defined; 1540)
[2023-10-18T10:42:41.782+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-18T10:42:41.809+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-18T11:17:14.969+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-18T11:17:14.981+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-18T11:17:14.981+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-18T11:17:14.994+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-17 00:00:00+00:00
[2023-10-18T11:17:15.000+0000] {standard_task_runner.py:57} INFO - Started process 361 to run task
[2023-10-18T11:17:15.003+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-17T00:00:00+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmp5rj82e9g']
[2023-10-18T11:17:15.004+0000] {standard_task_runner.py:85} INFO - Job 116: Subtask waiting_for_data
[2023-10-18T11:17:15.063+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [running]> on host 0e68aa52dfc7
[2023-10-18T11:17:15.152+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-17T00:00:00+00:00'
[2023-10-18T11:17:15.164+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-18T11:17:15.164+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-18T11:17:15.164+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-18T11:17:15.165+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-18T11:17:15.175+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210317T000000, start_date=20231018T111714, end_date=20231018T111715
[2023-10-18T11:17:15.223+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-18T11:17:15.253+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-19T12:11:50.691+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-19T12:11:50.703+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [queued]>
[2023-10-19T12:11:50.704+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-19T12:11:50.721+0000] {taskinstance.py:1382} INFO - Executing <Task(FileSensor): waiting_for_data> on 2021-03-17 00:00:00+00:00
[2023-10-19T12:11:50.730+0000] {standard_task_runner.py:57} INFO - Started process 406 to run task
[2023-10-19T12:11:50.737+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'copy_tranform_to_mongo', 'waiting_for_data', 'scheduled__2021-03-17T00:00:00+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/file_***-mongo.py', '--cfg-path', '/tmp/tmpiohg5toe']
[2023-10-19T12:11:50.739+0000] {standard_task_runner.py:85} INFO - Job 110: Subtask waiting_for_data
[2023-10-19T12:11:50.794+0000] {task_command.py:416} INFO - Running <TaskInstance: copy_tranform_to_mongo.waiting_for_data scheduled__2021-03-17T00:00:00+00:00 [running]> on host 6f5df7fea6fd
[2023-10-19T12:11:50.885+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='copy_tranform_to_mongo' AIRFLOW_CTX_TASK_ID='waiting_for_data' AIRFLOW_CTX_EXECUTION_DATE='2021-03-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-03-17T00:00:00+00:00'
[2023-10-19T12:11:50.896+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2023-10-19T12:11:50.896+0000] {filesystem.py:64} INFO - Poking for file /opt/in/tiktok_google_play_reviews.csv
[2023-10-19T12:11:50.897+0000] {filesystem.py:69} INFO - Found File /opt/in/tiktok_google_play_reviews.csv last modified: 20220405221612
[2023-10-19T12:11:50.897+0000] {base.py:287} INFO - Success criteria met. Exiting.
[2023-10-19T12:11:50.909+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=copy_tranform_to_mongo, task_id=waiting_for_data, execution_date=20210317T000000, start_date=20231019T121150, end_date=20231019T121150
[2023-10-19T12:11:50.952+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-19T12:11:51.006+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
